.github/workflows/build_dev_documentation.yml

```diff
@@ -39,6 +40,7 @@ jobs:
       - name: Set env
         run: |
           echo "WRITE=$(echo 'ghp_'$(wget -qO- lysand.re/doc-build-dev)'bm')" >> $GITHUB_ENV
+          node -v
```

Comment:
is this log useful? or was it here for debug reasons?

Reply:
Good catch, will remove it! It was for debugging earlier.